utils/helpers.py

```
@@ -0,0 +1,190 @@
+
+def sanitize_filename(filename: str) -> str:
+    """
+    Sanitize filename by removing invalid characters.
+    
+    Args:
+        filename: Original filename
+        
+    Returns:
+        Sanitized filename
+    """
+    import re
```

Pylint (3.3.7)

[convention] Import outside toplevel (re)

(C0415)